src/debug/di/rspriv.h

```diff
@@ -7560,7 +7569,7 @@ class CordbFunctionBreakpoint : public CordbBreakpoint,
                                 public ICorDebugFunctionBreakpoint
 {
 public:
-    CordbFunctionBreakpoint(CordbCode *code, SIZE_T offset);
+    CordbFunctionBreakpoint(CordbCode *code, SIZE_T offset, BOOL offsetIsIl = TRUE);
```

Comment:
CordbCode::CreateBreakpoint calls this when setting both native offset and IL offset breakpoints. Currently it uses the default value so it gets the native offset case wrong. I would eliminate the default value and ensure that each callsite is providing a correct value for the scenario it is handling.